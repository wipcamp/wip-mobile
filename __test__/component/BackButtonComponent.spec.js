/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import BackButton from '../../components/BackButtonComponent'

it('rendering', () => {
    const tree = renderer
        .create(<BackButton />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})