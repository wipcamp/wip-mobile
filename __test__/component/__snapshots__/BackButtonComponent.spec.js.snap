// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
      "paddingLeft": 10,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={false}
    android="md-arrow-round-back"
    ellipsizeMode="tail"
    ios="ios-arrow-back"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "color": "#000",
            "fontSize": 30,
          },
          Object {
            "color": "#000",
            "fontSize": 30,
          },
          Object {
            "color": "#FF8214",
          },
        ],
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
    virtual={undefined}
  >
    
  </Text>
</View>
`;
